### DISCUSSION TITLE
Facing 'difference between request time and server's time is too large'

### DISCUSSION THREAD

**Discussion by sridhar1495** [minio/minio#20796] [Q&A] [answered]
I am trying to access and store the object in MinIO from different machines which are in different locations and different time zone. Which some machines may not been in the correct time because of some reasons (example: manual override or some hardware crash). If i am trying to do the upload I am getting the 'difference between request time and server time is too large'. I know i can go and changing the time in those machines will resolve this problem. But in the customer who is going to access the MinIO, i cannot go and ask them to change the time, so only the required feature will work.
So, i am looking for the solution to disable this behavior. Please if there is any suggestion or configuration available for this scenario let me know.

**Comment by mshanmu** [comment, marked as answer]
I think there is a MinIO expectation that time skew cannot be large between machines. Setup NTP server and synchronize time in all the machines.

**Comment by klauspost** [comment]
Also, for features like versioning to work correctly, clocks must be synchronized.

**Comment by sridhar1495** [comment]
@mshanmu @klauspost Thanks, i understood and accept the points which you have provided. But the business telling that, the customer may or may not allow us to change the time in their machine. In this case if we have the option to disable this time check, doing this will be helpful to automate some of the things without depending on the customers. Can this be possible to do so?

**Reply by klauspost** [reply]
You cannot. We have no intention of making this a viable setup, since it will potentially break in so many ways.  How you can take "customers" without such basic knowledge is frankly beyond me, but I guess that is a gamble for them to discover.

**Reply by sridhar1495** [reply]
No problem. Just wanted to know whether it is possible to do or not. I will get back to the business and let them know what needs to be done. Thank you for the quick replies 👍
